BUILD

```diff
@@ -168,3 +168,17 @@ cc_library(
         "//google/cloud/bigquery:google_cloud_cpp_bigquery_mocks",
     ],
 )
+
+cc_library(
+    name = "bigquery",
```

Comment:
Do we want to get rid of the `//:experimental-bigquery` target now too? If not now, when do you think we should? I suspect we probably do want to, and we should include in the `CHANGELOG.md` that this API is now GA and references to `//:experimental-bigquery` will need to change.